config/Dockerfiles/tests.d/gcloud/06_gcloud_gce

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -xe
 
 # Verify the gcloud compute engine provisioning
-# distros.exclude: centos7 fedora28
+# distros.exclude: centos7
```

Comment:
Do we need to run this on centos7 and fedora30?

Reply:
Yes since its basic provisioning we do need to run it on fedora30 too.

Reply:
I meant to say fedora29 and fedora30 here.  But since its basic provisioning, why are we excluding any distros?